submissions/oluwatosinOke/phpAndDatabases/customerlist.php

```diff
@@ -0,0 +1,29 @@
+<?php
+require_once("cookie.php");
+    if ($result->rowCount()>0){
```

Comment:
This `$result` is what is always `return` each time you run the `$sql` query in the `cookie.php`. 

What we want is that the `$result` used here to either come from the `$sql`query (if cookie is not set) or from cookie (if cookie is set). You are already storing the data in a cookie called `view` ( see line 8 of cookie,php file) but that's all you are doing. You are not getting it or using it